config/manager/manager.yaml

```diff
@@ -61,4 +61,6 @@ spec:
             value: "default@test.com"
           - name: QUOTA
             value: "200"
+          - name: DEFAULT_ADDON_PARAMETERS_SECRET_NAME
+            value: "addon-managed-api-service-parameters"
```

Comment:
@makslion I wonder would it not be better to expose this envar via subscription config in MTR? 
By doing so, we would have good enough flexibility to set this up as `addon-managed-api-service-parameters` for rhoam and `addon-managed-api-service-internal-parameters` for edge evn.

Reply:
this envar is needed for local and bundles install types - would we need it to be `addon-managed-api-service-internal-parameters`? :thinking: And by putting it in the MTR wouldn't we leave these instal types without a default value?